backend/src/v5/schemas/tickets/index.js

```diff
@@ -90,6 +90,9 @@ const generatePropertiesValidator = async (teamspace, properties, oldProperties,
 				}
 
 				const oldValue = oldProperties?.[prop.name];
+
+				validator = validator.test('Immutable property', `Immutable property ${prop.name} cannot be edited`, (value) => !(prop.immutable && oldValue && value));
```

Comment:
Can we wrap it around with ifs like the other stuff?

also we should check for undefined as it's possible for oldValue to be `false` or `0` if the type is boolean/number

```suggestion
if(prop.immutable && oldValue !==undefined){
				validator = validator.test('Immutable property', `Immutable property ${prop.name} cannot be edited`, (value) => value !== undefined);
}
```